src/App.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import FlipCounter from "./FlipCounter";
+
+import "./App.css";
+class App extends Component {
+  constructor(props) {
```

Comment:
Composition means, that you divide your application to a bunch of components, each of one having a separate zone of responsibility.

So you need to use a separate component to store the user's inputs value and render this input.